research/object_detection/g3doc/dataset_explorer.md

```diff
@@ -0,0 +1,20 @@
+# Dataset Explorer
+
+Prior to training, you can explore your dataset with the Dataset Explorer
+script.
+
+<p align="center">
+  <img src="img/dataset_explorer.png" width=676 height=367>
+</p>
+
+```bash
+# Explore the first 100 data points.
+python object_detection/dataset_explorer/dataset_explorer.py \
```

Comment:
Seems this script is missing. Should it be added, or is this README premature?

Reply:
Thanks for catching this! Removed this premature readme file.